ADOPTERS.md

```diff
@@ -1,15 +1,25 @@
 ## Strimzi adopters
 
+Are you currently using Strimzi in production?
+Do you have products based on Strimzi or provide Strimzi-related services?
+Please let us know by adding your company name and, if you want, a description of your use case to this document!
+
+### Vendors
+
 * [Axual](https://axual.com/)
     * [Optimizing Kafka Cluster Deployments in Kubernetes](https://itnext.io/optimizing-kafka-cluster-deployments-in-kubernetes-ceda3f95c157)
+* [Cloudera](https://www.cloudera.com/products/stream-processing.html)
+* [IBM](https://www.ibm.com/cloud/event-streams)
```

Comment:
```suggestion
* [IBM](https://www.ibm.com/products/event-streams)
```

To be fair, the old URL does redirect to this, but while we're updating it'd be good to refresh this, please.